policies-and-procedures/jellyfin-constitution.md

```diff
@@ -0,0 +1,156 @@
+
+2. Subproject Leaders are responsible for the day-to-day operations of their sub-project and the handling of technical and managerial decisions for the sub-project. For the purposes of all decisions pertaining to the sub-project exclusively, the Subproject Leader shall be responsible for the final decision, in consultation with the Project Leader if necessary.
+
+3. Subproject Leaders shall have Admin privileges to the sub-project repositories they manage and have the power to assign permissions on the sub-project to other team members, for instance merge permissions, at their discretion.
+
+4. Subproject Leaders should seek consensus wherever possible with other contributors to the sub-project, however they are empowered to make decisions as required. Disputes may be escalated to the Leadership Team or Project Leader if desired for further decision and resolution. As outlined in power #2, the Subproject Leader shall have the final say in any decision reached here.
+
+#### Procedure
+
+1. The initial Subproject Leader for a sub-project shall be determined depending on the source of the sub-project.
+
+     i. For Subproject that originate among the existing Jellyfin Project, for instance split-out libraries, or are forks of other projects, the sub-project leader shall be determined by the Project Leader in consultation with the Leadership Team at the time the project is integrated into the Jellyfin Organization.
+
+    ii. For sub-projects that originate as the sole work of an existing Jellyfin Team member, that person shall become Subproject Leader at the time the project is transferred to the Jellyfin Organization.
+
+   iii. For sub-projects that originate as the sole work of an outside contributor who desires to make the project a sub-project of the Jellyfin Organization, that person shall become a member of the Jellyfin Team, subject to standard nomination rules for Team Members, and shall become Subproject Leader at the time the project is transferred to the Jellyfin Organization. If nomination of the member fails for any reason, the project shall not be integrated into the Jellyfin Organization and shall remain a 3rd-party project.
+
+    iv. For sub-projects that, at the time of the adoption of this condition, do not already have a clear Subproject Leader, elections shall be held by the existing contributors to select the Subproject Leader. The method of election shall be ranked voting instant runoff until one candidate has secured an absolute majority of the vote.
+
+2. There may be either one Subproject Leader, or two Subproject Co-Leaders for larger sub-projects, for any given sub-project. Co-Leaders shall share all powers but should strive for consesnsus decisions.
```

Comment:
```suggestion
2. There may be either one Subproject Leader, or two Subproject Co-Leaders for larger sub-projects, for any given sub-project. Co-Leaders shall share all powers but should strive for consensus in all decisions.
```

Reply:
Both work, in this case I'm using "consensus decisions" as a single noun, i.e. "decisions that are agreed on by consensus".

Reply:
Oh I see. However, the typo is still there :)